test/testUcumLhcUtils.spec.js

```diff
@@ -222,68 +221,404 @@ it("should return a message for invalid unit strings", function() {
+    var resp = utils.convertUnitTo('g', 39.09, 'eq', {
+      charge: 1,
+      molecularWeight: 39.09
+    });
+    assert.equal(resp.status, 'succeeded', resp.status + resp.msg);
+    assert.equal(resp.toVal.toPrecision(1), 1, resp.toVal.toPrecision(1));
+  });
+
+  /**
+   * CaCl2 -> Ca<sup>2+</sup> + 2 Cl-
+   * Ca++ has a valence of 2 and a molecular weight of 40.08
+   * ∴ 1 mol of Ca = 2 eq of Ca
+   * Equivalent mass of Ca++ is its atomic mass divided by valence 2.
+   * Mass of calcium is 40.08 g/mole, so its equivalent mass is 40.08/2 =  20.04 g/equiv.  
+   * But we are converting 40.08 g to eq, so the result should be 40.08g/20.04g = 2 eq
+   */
+  it("should return 2 for a request to convert 40.08 g to eq with charge 2 and mw 40.08", function() {
+    var resp = utils.convertUnitTo('g', 40.08, 'eq', {
+      charge: 2,
+      molecularWeight: 40.08
+    });
+    assert.equal(resp.status, 'succeeded', resp.status + resp.msg);
+    assert.equal(resp.toVal.toPrecision(1), 2, resp.toVal.toPrecision(1));
+  });
+
+  it("should return 2 for a request to convert 40.08 g to eq with charge 2 and mw 40.08", function() {
+    var resp = utils.convertUnitTo('g', 40.08, 'meq', {
+      charge: 2,
+      molecularWeight: 40.08
+    });
+    assert.equal(resp.status, 'succeeded', resp.status + resp.msg);
+    assert.equal(resp.toVal.toPrecision(1), 2000, resp.toVal.toPrecision(1));
+  });
+
+  /** Eq to mol conversion tests
+   * 1 equivalent equals 1/v moles, where v is the valency factor (=== valence for an element)
+   * if valence is 1, then 1 equivalent equals 1 mole
+   * if valence is 2, then 1 equivalent equals 0.5 moles
+   * if valence is 3, then 1 equivalent equals 0.33 moles
+   * etc. 
+   */
+
+  // mw should be ignored when converting between eq and mol
+  it("should return 0.33 for a request to convert 1 eq/L to mol/L with charge 3 and mw 1000", function() {
+    var resp = utils.convertUnitTo('eq/L', 1, 'mol/L', {
+      charge: 3,
+      molecularWeight: 1000
+    });
+    assert.equal(resp.status, 'succeeded', resp.status + resp.msg);
+    assert.equal(resp.toVal.toPrecision(2), 0.33, resp.toVal.toPrecision(2));
+  });
+
+  // Should throw an error if no charge is provided for a conversion to mol since charge is needed for the calculation
+  it("should return error for a request to convert eq to mol with no charge and mw 40.08", function() {
+    var resp = utils.convertUnitTo('eq', 1, 'mol', {
+      molecularWeight: 40.08,
+    });
+    assert.equal(resp.status, 'failed', resp.status + resp.msg);
+    assert.equal(resp.msg[0], Ucum.needEqChargeMsg_,
+      resp.msg[0]);
+    assert.equal(resp.toVal, null, resp.toVal);
+    assert.equal(resp.fromUnit, undefined, resp.fromUnit);
+    assert.equal(resp.toUnit, undefined, resp.toUnit);
+  });  
+
+  it("should return 1 for a request to convert 1 eq/L to mol/L with charge 1", function() {
```

Comment:
These five tests seem to be redundant.  What more do they test than was tested above?  See if you can pare down the number.